ground/src/main/java/com/google/android/ground/persistence/local/room/LocalDatabase.kt

```diff
@@ -89,6 +90,7 @@ import com.google.android.ground.persistence.local.room.fields.TileSetEntityStat
     ],
   version = Config.DB_VERSION,
   exportSchema = true,
+  autoMigrations = [AutoMigration(from = 120, to = 121)],
```

Comment:
Thanks for pointing it out, it doesn't allow app to auto-migrate from 119 to 121. Will try to find a fix for this.

Reply:
One solution could be to go back in history and add schemas for all versions which we think people are still using. Then we can add similar auto-migrations for each upgrade and it should work fine. Wdyt?

Reply:
Since we currently know all our users, we could also ask them to make sure they sync all data before updating, and use destructive migrations this one time. Wdyt?

Reply:
Yes, we can ask them to sync all data, uninstall the app and then install the updated version.

Reply:
Actually, even better - it looks like the app is currently inactive. I just pinged the chat to see if data collection is paused, if so we can just ask everyone to uninstall and reinstall just  for the next release.

Reply:
Now the code ensures that we don't allow the db updates without a migration strategy. So, this should be the only instance where it needs to be done.